rotkehlchen/db/drivers/gevent.py

```diff
@@ -506,7 +506,7 @@ def schema_sanity_check(self) -> None:
         db_name = self.connection_type.name.lower()
         with self.read_ctx() as cursor:
             cursor.execute('SELECT name, sql FROM sqlite_master WHERE type="table"')
-            tables_data_from_db = {}
+            tables_data_from_db: dict[str, tuple[str, str]] = {}
             for (name, raw_script) in cursor:
                 minimized_script = raw_script.replace(' ', '').replace('\n', '').replace('\'', '"')
```

Comment:
Hmm if to do case-insensitive comparisons, then maybe it will be better to have entire `minimized_schema.py` files lowercase and do
```suggestion
                minimized_script = raw_script.replace(' ', '').replace('\n', '').replace('\'', '"').lower()
```

here ? What do you think?